nextcloud_install_production.sh

```diff
@@ -17,9 +17,30 @@ fi
 # shellcheck disable=2034,2059
 true
 SCRIPT_NAME="Nextcloud Install Script"
+SCRIPT_EXPLAINER="Welcome to the Nextcloud VM installer.
+It simplifies the installation of Nextcloud and its integrations by a lot and has other great features.
+You can find the sourcecode and other advices here: https://github.com/nextcloud/vm"
 # shellcheck source=lib.sh
 source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
 
+# Show explainer
+any_key "$SCRIPT_EXPLAINER\nPress any key to continue."
+
+if [ "no" = $(ask_yes_or_no "Do you want to continue with this script?") ]
+then
+    exit 1
+fi
+
+# Let the user run an initial packet update
+if [ "no" = $(ask_yes_or_no "Do you want to run an initial packet update?") ]
```

Comment:
What would be a better question to not make running an initial packet update the default?